### ISSUE TITLE
[Wishlist]: Separate long doc into tabs in the editor

### ISSUE THREAD

**Issue by runsun-edu** [iodide-project/iodide#1622]
As a doc gets too long, it's hard to find where you want to edit, and takes a lot of time scrolling.

It'd be good if a long doc can be separated --- behind the scene  --- into tabbed sub-docs, then automatically assembled into a whole doc upon rendering. 

With that feature, it'd be also much easier for users if relocation of a section is needed --- just drag the tab and drop it between other tabs.

**Comment by bcolloran** [comment]
yep, it's a great idea @runsun-edu! something we have talked about and wanted, thank you for giving us a nudge, it'll help us prioritize it! :-)
